firebase/initFirebase.js

```diff
@@ -1,15 +1,35 @@
 /*
 This module initializes a firebase 'app' instance.
 */
-import * as firebase from 'firebase';
+const firebase = require('firebase');
+require("firebase/firestore");
+
+
+// // Initialize Firebase:
+// firebase.initializeApp({
+//   apiKey: process.env.FIREBASE_API_KEY,
+//   authDomain: process.env.FIREBASE_AUTH_DOMAIN,
+//   databaseURL: process.env.FIREBASE_DATABASE_URL,
+// });
+
+// Initialize Firebase
+const config = {
+  apiKey: "AIzaSyA3Gjq9-4MuPFauhHzk6pGAjoQMC4I4dz0",
+  authDomain: "codesnippets-c9eee.firebaseapp.com",
+  databaseURL: "https://codesnippets-c9eee.firebaseio.com",
+  projectId: "codesnippets-c9eee",
+  storageBucket: "codesnippets-c9eee.appspot.com",
+  messagingSenderId: "846625889305"
+};
```

Comment:
I don't think this is good practice.
`https://medium.freecodecamp.org/how-to-securely-store-api-keys-4ff3ea19ebda`